src/components/Logout.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <div/>
+    <div />
```

Comment:
Let's disable the git autoformatting?... None of these formats are relevant to the actual change

Reply:
@pandafox git autoformatting is not enabled. I manually ran `npm run lint-fix` to fix the lint warnings.